regen/feature.pl

```diff
@@ -392,14 +392,7 @@ sub longest {
 
 #define CLEARFEATUREBITS() (PL_compiling.cop_features = 0)
 
-#define STOREFEATUREBITSHH(hh) \\
-  (hv_stores((hh), "feature/bits", newSVuv(PL_compiling.cop_features)))
-
-#define FETCHFEATUREBITSHH(hh)                              \\
-  STMT_START {                                              \\
-      SV **fbsv = hv_fetchs((hh), "feature/bits", FALSE);   \\
-      PL_compiling.cop_features = fbsv ? SvUV(*fbsv) : 0;   \\
-  } STMT_END
+#define FETCHFEATUREBITSHH(hh) S_fetch_feature_bits_hh(aTHX_ (hh))
```

Comment:
This commit turned 1 `hv_fetch()`, into 26 always failing `hv_fetch()` calls, on a `HV*` with only 1 key/1 HE* in it, with a key name of "CORE/prevailing_version".